nuxt.config.ts

```diff
@@ -1,4 +1,28 @@
 // https://nuxt.com/docs/api/configuration/nuxt-config
+import type { LocaleObject } from '@nuxtjs/i18n'
+
+const processLocales = () => {
+  const locales = process.env.NUXT_PUBLIC_DATO_REGION_LOCALES
+  if (locales) {
+    const localeArray = locales?.split(',')
```

Comment:
To ensure accidental whitespaces around the locale are removed:

```suggestion
    const localeArray = locales?.split(',').map(l => l.trim())
```